app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  def update_user_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
+
```

Comment:
- [OPTIONAL] Good job so far but I think it would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_user_posts_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private 😁 😁